Use cargo auditable to build the binary and disable dependency downloading

diff --git a/a/build-aux/cargo_build.py b/b/build-aux/cargo_build.py
index 5fca7ef2719..5fd7466f800 100644
--- a/build-aux/cargo_build.py
+++ b/build-aux/cargo_build.py
@@ -22,7 +22,7 @@ print(f"""
     output_file: {output_file}
 """, file=sys.stderr)
 
-cargo_call = f"env {cargo_env} {cargo_cmd} build {cargo_options}"
+cargo_call = f"env {cargo_env} {cargo_cmd} auditable build --frozen {cargo_options}"
 cp_call = f"cp {bin_output} {output_file}"
 
 print(cargo_call, file=sys.stderr)
